documentation/source/conf.py

```diff
@@ -0,0 +1,65 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+from infertrade import _version
+
+sys.path.insert(0, os.path.abspath('../../infertrade/'))
```

Comment:
```suggestion

sys.path.insert(0, os.path.abspath('../../infertrade/'))
import _version
```

Two things here:
(a) what I had thought of doing when I suggested inserting the `from infertrade import _version` line was simply to import the `_version.py` file that sits at the base of the infertrade package - which itself would, thanks to the `sys.path.insert` line, be placed at the beggining of the import path... however, this suggestion itself was incorrect, since the line would actually have to be just `import _version` (the `from infertrade` part would only be correct if the directory inserted in the path was the one *immediately above* the package);
(b) also, for the reasons previously indicated, the `sys.path.insert` line would have to sit above the the `import _version` line for this latter line to work (alternatively, we could just copy the `get_version` function from the `setup.py` file, as ta-oliver suggested in https://github.com/ta-oliver/infertrade/pull/103#issuecomment-852164432 ... @ta-oliver , do you have any preferences with regards to this?)

Reply:
No strong preference - whichever would be easiest to understand and maintain?

Reply:
@fhashim , just so you're aware, I'm committing this change.